src/revlanguage/moves/mixture/Move_MixtureAllocation.h

```diff
@@ -51,7 +51,7 @@ namespace RevLanguage {
 #include "ArgumentRules.h"
 #include "MetropolisHastingsMove.h"
 #include "ModelVector.h"
-#include "MixtureAllocationProposal.h"
+#include "../../../core/moves/mixture/OrderedEventSlideProposal.h"
```

Comment:
I don't think the full-path is needed here, unless there are multiple different `OrderedEventSlideProposal.h` files...

Reply:
And if there are... maybe the `../../../` is not necessary?

Reply:
This was actually erroneous... I switched it back to `MixtureAllocationProposal.h`